index.html

```diff
@@ -41,6 +45,177 @@ <h1 class="headline-heading">I'm Usman <br> Glad to see you!</h1>
+            <p class="paragraph">CANOPY</p>
+            <i class="fa-solid fa-circle works-card-client-counter"></i>
+            <p class="works-card-client-role paragraph">Back End Dev</p>
+            <i class="fa-solid fa-circle works-card-client-counter"></i>
+            <p class="works-card-client-year paragraph">2015</p>
+        </section>
+        <p class="paragraph works-card-paragraph">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <section class="works-card-tags">
+            <p class="works-card-tag">html</p>
+            <p class="works-card-tag">css</p>
+            <p class="works-card-tag">javascript</p>
+        </section>
+        <button>See Project</button>
+    </section>
+
+    <!-- Card 3 Ends -->
+    <!-- Card 4 starts -->
+
+    <section class="works-card">
+        <img src="./assets/works-4.png" alt="Recent Work">
+        <h1 class="works-card-heading-large">Multi-Post Stories</h1>
+        <section class="works-card-client">
+            <p class="paragraph">CANOPY</p>
+            <i class="fa-solid fa-circle works-card-client-counter"></i>
+            <p class="works-card-client-role paragraph">Back End Dev</p>
+            <i class="fa-solid fa-circle works-card-client-counter"></i>
+            <p class="works-card-client-year paragraph">2015</p>
+        </section>
+        <p class="paragraph works-card-paragraph">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <section class="works-card-tags">
+            <p class="works-card-tag">html</p>
+            <p class="works-card-tag">css</p>
+            <p class="works-card-tag">javascript</p>
+        </section>
+        <button>See Project</button>
+    </section>
+
+    <!-- Card 4 Ends -->
+    <!-- Works Section Ends Here -->
+    <!-- About Section Starts Here -->
+
+    <section class="about">
+        <section class="headline">
+            <h1 class="headline-heading heading-large">About <br> Myself</h1>
+            <p class="headline-paragraph paragraph">I’m a software developer! I can help you build a product , feature
+                or website
+                Look
+                through some of my work
+                and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+                me.
+            </p>
+            <p class="headline-sm-suport">LET’S CONNECT</p>
+            <ul class="social-icons">
+                <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-facebook"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-linkedin"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-medium"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
+            </ul>
```

Comment:
Good job in here, but as you see the size of social media icons is a bit smaller in your design, kindly make sure to increase it to make it more similar to the one in Figma.

![image](https://user-images.githubusercontent.com/34779928/212386282-2489bf0d-8ac8-4e9b-a285-9981c4798107.png)